### PR TITLE
Rewrite russian localization

### PR DESCRIPTION
**Pull request by Fogsight**
Auto-translated text is quite horrible. Fixed version.

### CODE REVIEW

**Comment by WardBenjamin** [comment]
I can't read Russian, but looks great! Thanks for the contribution!

**Comment by quill18** [comment]
@WardBenjamin Do you have the ability to merge PR's yet?  I think you should.

**Comment by vogonistic** [comment]
@quill18 He didn't 2 hours ago. I can't either.

File: Assets/StreamingAssets/Localization/ru_RU.lang [hunk 1]
```diff
@@ -1,31 +1,35 @@
...
-okay=Хорошо
-cancel=Отмена
-filename=Имя файла
-selection_hp={0} / {1}
-build=Сборки {0}
-furn_steel_wall=Стальная стена
-furn_steel_wall_desc=Стены сделаны из стали.
+temp_pathfinding_test=Навигационный Тест
+selection_panel_title=Текущий Выбор
+save_file=Файл Сохранения...
```
Comment (Grenkin1988) [inline Assets/StreamingAssets/Localization/ru_RU.lang]: Maybe we shoulв have everywhere not just Save File... but - Save File(s)... ?
Is so - than it should be Файл(ы) Сохранения...

Comment (Grenkin1988) [inline Assets/StreamingAssets/Localization/ru_RU.lang]: This is header of save file window. This probably should be Сохранить файл...

**Pull request merged**